pages/Guides/supernode-program/docs.en.md

```diff
@@ -8,90 +8,89 @@ taxonomy:
+- **Step 5**: Shut down the NIS node, if running.
 
-  To find your delegated private key please use NCC and go to *View Account Details* and then *Show Remote Account's PRIVATE Key*.
+- **Step 6**: Configure your NIS node to autoboot with the **delegated** private key.
 
   You will need to edit the config file in your NIS folder to do this. Here is some help:
 
-  When you unzipped the standalone you made a folder called ``package``. Open that folder and double click on the folder ``nis`` and then open the file ``config.properties`` with a text editor. You will then need to find line 43 which says ``#nis.bootKey =``. Now delete the "#" so it only says  ``nis.bootKey =``. Next delete ``#0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef`` and replace it with your delegate private key you just made, and again don't keep the "#".
+  When you unzipped the standalone you made a folder called ``package``. Open that folder and double-click on the folder ``nis`` and then open the file ``config.properties`` with a text editor. You need to find line which says ``#nis.bootKey =``. Now delete the "#" so it only says ``nis.bootKey =``. Next delete ``#0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef`` and replace it with your delegate private key from Step 3.
 
-  Next in line 44, ``#nis.bootName``, remove the "#" and then erase "foobar" and instead write the name of your node. Any name is fine. Lastly, set ``nis.shouldAutoHarvestOnBoot`` to true.
+  In the next line (``#nis.bootName``), remove the "#" and then erase "foobar" and instead write the name of your node. Any name is fine. Lastly, make sure that ``nis.shouldAutoHarvestOnBoot`` is set to ``true``.
 
-  ![nis config.proprties](Screenshot-2016-02-07-05-25-08.png)
+  Optionally (node will try to autodetect IP automatically if not defined), you can define your domain or IP using ``nem.host`` property.
 
-  Save your edits and close the text editor.
+  ![nis config.properties](Screenshot-2016-02-07-05-25-08.png)
 
-  Please review this [tutorial](http://nem.ghost.io/nis-auto-start-and-auto-harvest/) if you need more help.
+  Save your edits and close the text editor.
 
-  To test if you have done this correctly restart your standalone NIS and NCC. If you have followed all steps correctly, you will see that your NIS is booting and synchronizing automatically, even if you didn't log into your wallet yet.
+  To test if you have done this correctly restart your node. If you have followed all steps correctly, you will see that your NIS is booting and synchronizing automatically.
 
-- **Step 6**: Download the [node servant program](https://drive.google.com/open?id=0B_0Z9jDGHPxPVmdCVDFTT25CXzg).
+- **Step 7**: Download the [node servant](https://bob.nem.ninja/servant_0_0_4.zip).
   
   This is a very light application that will run 24/7 on the same machine as your NIS to perform tests and monitor the network.
```

Comment:
Change `on the same machine as your NIS` to `on the same machine as your NIS client`